cmd/lakectl/cmd/root.go

```diff
@@ -521,6 +538,10 @@ func initConfig() {
 	// set defaults
 	viper.SetDefault("metastore.hive.db_location_uri", "file:/user/hive/warehouse/")
 	viper.SetDefault("server.endpoint_url", "http://127.0.0.1:8000")
+	viper.SetDefault("server.retries.enabled", true)
```

Comment:
Isn't the number of retries enough to decide if we have retries or not?

Reply:
It is, but I'd prefer to be explicit if it doesn't cost me anything.

Reply:
Doesn't cost you, but does cost the user understanding
What happens if I don't want reties and I change retries to 0 because that's what I'm used to, what will happen here?